### PR TITLE
Add detail to Southampton Kickstart event

### PR DESCRIPTION
**Pull request by Adimote**
Add some extra details to the Southampton kickstart event:
- Add an Address
- Flesh out getting to campus page, as the anchor link doesn't work.
- Mention we'll have computers available.

### CODE REVIEW

File: _events/sr2019/southampton-kickstart.md [hunk 1]
```diff
@@ -12,13 +12,25 @@ be announced and robotics kits will be handed out to the teams.
...
 ## Directions
 
-* How to [get to campus][soton-campus-directions]
+The Southampton kickstart is being held at the University of Southampton at:
+
+EEE Building (32),
+University of Southampton,
+University Road,
+Southampton,
+SO17 1BJ
```
Comment (PeterJCLaw) [inline _events/sr2019/southampton-kickstart.md]: You'll need to add two trailing spaces on these lines if you don;t want it to automatically wrap into a paragraph.
```suggestion
EEE Building (32),  
University of Southampton,  
University Road,  
Southampton,  
SO17 1BJ  
```

**Comment by PeterJCLaw** [comment]
Note: I've not run these changes - I would strongly recommend you do so before merging.

**Comment by Adimote** [comment]
fixed, please re-review

File: _events/sr2019/southampton-kickstart.md [hunk 2]
```diff
@@ -12,13 +12,25 @@ be announced and robotics kits will be handed out to the teams.
...
-* How to [get to campus][soton-campus-directions]
+The Southampton kickstart is being held at the University of Southampton at:
+
+EEE Building (32),  
+University of Southampton,  
+University Road,  
+Southampton,  
+SO17 1BJ  
+
+For Public transport or driving instructions, see the University of Southampton's information on [getting to our campuses][soton-campus-directions] (scroll down to Highfield campus).
```
Comment (PeterJCLaw) [inline _events/sr2019/southampton-kickstart.md]: ah, just spotted that "Public" here should probably be lower case.
```suggestion
For public transport or driving instructions, see the University of Southampton's information on [getting to our campuses][soton-campus-directions] (scroll down to Highfield campus).
```

Comment (Adimote) [reply _events/sr2019/southampton-kickstart.md]: I really like this 'suggested changes' feature :+1:

**Comment by Adimote** [comment]
Forgot to merge until just now >.>

**Pull request merged**